test/mock_batch_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  String users = "users";
+  MockFirestoreInstance firestore = MockFirestoreInstance();
+
+  Map<String, dynamic> userDocumentMock(String userId) {
+    Map<String, dynamic> data = {
+      "id": userId,
+      "updatedAt": DateTime.now(),
+      "createdAt": DateTime.now(),
+    };
+    return data;
+  }
+
+  Future<List<Map<String, dynamic>>> createUsers({int times}) async {
```

Comment:
This execute synchronously, so the return type should simply be `List<Map<String, dynamic>>`:

```dart
List<Map<String, dynamic>> createUsers({int times}) {
  ...
}
```